### PR TITLE
add job & script to check translations

### PR DESCRIPTION
**Pull request by tenzap**
For now it is normal the translation check fails because the other PR containing everything are not merged.

The check_translation job is not mandatory to pass. Thus it is not marked as "required" in the list of checks bellow.
However, I would prefer that if it doesn't pass (but the 2 others do pass) to have a green tick, but couldn't find how to do so.

### CODE REVIEW

File: check_translation.php [hunk 1]
```diff
@@ -0,0 +1,595 @@
+#!/usr/bin/php
```
Comment (kingster) [inline check_translation.php:1]: let's move these helper files inside the `scripts/helpers` directory.

Comment (tenzap) [reply check_translation.php:1]: `scripts` if for kalkun scripts, not for devel or helper scripts.
We can add them in another toplevel dir, but I prefer not into `scripts`.

Comment (kingster) [reply check_translation.php:1]: That also works, please create one. Let's not keep these in root dir.

Comment (tenzap) [reply check_translation.php:1]: Any suggestion for a short name?

Comment (tenzap) [reply check_translation.php:1]: I think `utils` and `docs` would be fine with files placed in it as suggested in my previous comment. Are you ok with that?

Comment (kingster) [reply check_translation.php:1]: Sounds good to me 👍

**Review by kingster — DISMISSED** [review]

File: check_translation.php (hunk 1 above)
Comment (tenzap) [reply check_translation.php]: Done for this script.
Other are in https://github.com/kalkun-sms/Kalkun/pull/386
Please review again.

**Comment by tenzap** [comment]
oh seems your approval got lost by a forced push...

**Review by kingster — APPROVED** [review]

**Comment by tenzap** [comment]
Any idea on how to achieve this?

> However, I would prefer that if it doesn't pass (but the 2 others do pass) to have a green tick (globally), but couldn't find how to do so.

**Pull request merged**